source/includes/_documentation.md

```diff
@@ -4854,6 +4854,194 @@ The angular Button:
 
 #### <a href="https://github.com/jdi-testing/jdi-light/blob/master/jdi-light-angular-tests/src/test/java/io/github/epam/angular/tests/elements/common/BasicButtonTests.java" target="_blank">Basic Button java tests examples</a>
 ---
+## Angular Complex elements 
+
+### List 
+
+#### <a href="https://material.angular.io/components/list/overview" target="_blank">List overview</a>
+
+List is a container component that wraps and formats a series of line items.
+
+There is two different lists in Angular: Basic list and List with sections.
```

Comment:
There seems to be a typo: "There is" -> "There are"

Reply:
Yes. Changes added.